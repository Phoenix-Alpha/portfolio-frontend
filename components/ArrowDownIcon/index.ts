export * from './ArrowDownIcon'
export { default } from './ArrowDownIcon'
