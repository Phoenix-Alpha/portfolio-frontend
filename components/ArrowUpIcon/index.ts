export * from './ArrowUpIcon'
export { default } from './ArrowUpIcon'
