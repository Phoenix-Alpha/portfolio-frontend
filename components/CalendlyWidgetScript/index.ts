export * from './CalendlyWidgetScript'
export { default } from './CalendlyWidgetScript'
