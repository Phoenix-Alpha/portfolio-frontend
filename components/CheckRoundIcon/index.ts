export * from './CheckRoundIcon'
export { default } from './CheckRoundIcon'
