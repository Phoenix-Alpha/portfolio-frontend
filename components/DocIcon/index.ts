export * from './DocIcon'
export { default } from './DocIcon'
