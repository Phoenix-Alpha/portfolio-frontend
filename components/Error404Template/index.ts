export * from './Error404Template'
export { default } from './Error404Template'
