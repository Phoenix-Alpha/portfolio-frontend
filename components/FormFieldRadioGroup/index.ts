export * from './FormFieldRadioGroup'
export { default } from './FormFieldRadioGroup'
