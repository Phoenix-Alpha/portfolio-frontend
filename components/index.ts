// Logos & Icons
export * from './SiteLogo'
export * from './CloseIcon'
export * from './CheckIcon'
export * from './CheckRoundIcon'
export * from './ArrowDownIcon'
export * from './ArrowUpIcon'
// Layouts
export * from './Col'
export * from './Row'
export * from './Container'
// Basic components
export * from './Divider'
export * from './Popover'
export * from './Modal'
export * from './ShadowBackdrop'
// Buttons
export * from './Button'
export * from './CloseButton'
export * from './MenuToggleButton'
export * from './PopoverButton'
// Links
export * from './ButtonLink'
export * from './PlainLink'
// Form
export * from './Input'
export * from './FormField'
export * from './FormFieldError'
export * from './Radio'
export * from './RadioGroup'
export * from './ToggleMenuIcon'
export * from './FormFieldRadioGroup'
export * from './FormFieldCheckbox'
export * from './Checkbox'
export * from './FormFieldTextArea'
export * from './FormSelect'
// Layout elements
export * from './MainNavItem'
export * from './MainNav'
export * from './MainNavDropdownMenuItem'
export * from './MainNavDropdownMenu'
export * from './FooterMenu'
export * from './SiteHead'
export * from './SiteHeader'
export * from './SiteFooter'
export * from './SiteLayout'
// Misc
export * from './CircularProgressBar'
export * from './CalendlyWidgetScript'
// Sections
export * from './HomeHeroSection'
// Templates
export * from './ErrorTemplate'
export * from './Error500Template'
export * from './Error404Template'
export * from './HomePageTemplate'
export * from './MaintenanceTemplate'
