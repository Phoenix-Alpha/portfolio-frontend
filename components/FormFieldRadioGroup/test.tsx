import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Radio } from '@/components'

import { FormFieldRadioGroup } from '.'

describe('<FormFieldRadioGroup />', () => {
  const name = 'test-name'
  const radio1 = <Radio value={1}>Test Radio1</Radio>
  const radio2 = <Radio value={2}>Test Radio2</Radio>

  it('should renders correct <FormFieldRadioGroup />', () => {
    const wrapper = shallow(
      <FormFieldRadioGroup name={name}>
        {radio1}
        {radio2}
      </FormFieldRadioGroup>
    )

    expect(wrapper.find('Field').prop('name')).toMatch(name)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
