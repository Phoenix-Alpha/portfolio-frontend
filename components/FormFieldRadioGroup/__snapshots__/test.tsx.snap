// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormFieldRadioGroup /> should renders correct <FormFieldRadioGroup /> 1`] = `
<Field
  name="test-name"
>
  <Component />
</Field>
`;
